pkg/kn/commands/service/apply_mock_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package service
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+	"gotest.tools/assert"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	servingv1 "knative.dev/serving/pkg/apis/serving/v1"
+
+	knclient "knative.dev/client/pkg/serving/v1"
+	"knative.dev/client/pkg/util/mock"
+	"knative.dev/client/pkg/wait"
+
+	"knative.dev/client/pkg/util"
+)
+
+func TestServiceApplyCreateMock(t *testing.T) {
+	// New mock client
+	client := knclient.NewMockKnServiceClient(t)
+
+	//service := createServiceWithImage("foo", "gcr.io/foo/bar:baz")
```

Comment:
Is this intended or forgotten? The commented line.

Reply:
it's a left over from the test development. will clean up, good find